pkg/api/resolver_mutation_site.go

```diff
@@ -0,0 +1,109 @@
+package api
+
+import (
+	"context"
+	"time"
+
+	"github.com/gofrs/uuid"
+
+	"github.com/stashapp/stash-box/pkg/models"
+)
+
+func (r *mutationResolver) SiteCreate(ctx context.Context, input models.SiteCreateInput) (*models.Site, error) {
+	if err := validateAdmin(ctx); err != nil {
+		return nil, err
+	}
+
+	var err error
```

Comment:
Can be removed, replaced by `@hasRole(role: ADMIN)` directive on schema.
```suggestion
	var err error
```